shell/platform/tizen/flutter_tizen_engine.cc

```diff
@@ -91,6 +91,7 @@ FlutterTizenEngine::FlutterTizenEngine(const FlutterProjectBundle& project)
 }
 
 FlutterTizenEngine::~FlutterTizenEngine() {
+  StopEngine();
   renderer_ = nullptr;
```

Comment:
We shouldn't need to set pointers to null in a class destructor.

Reply:
Thanks. I fixed it.